lib/ddr/models/file_management.rb

```diff
@@ -1,198 +1,195 @@
-require "ddr-antivirus"
```

Comment:
The change above is the only significant one in the file.  The rest is caused by collapsing separate modules into `Ddr::Models` and resulting changes in spacing.